app/tests/e2e/test_credentials.py

```diff
@@ -312,3 +312,70 @@ async def test_issue_credential_with_save_exchange_record(
         await faber_client.delete(f"{CREDENTIALS_BASE_PATH}/{cred_ex_id}")  # Clean up
     else:
         assert len(faber_cred_ex_recs) == 0  # default is to remove records
+
+
+@pytest.fixture(scope="function")
+async def issue_revocable_credentials_to_alice_and_revoke(
```

Comment:
We can potentially parameterise this test with a boolean param: `batch_revoke: [True, False]`
And then in the last step where faber revokes each of the 3 in the for loop:
```
# revoke all credentials in list
    for cred in cred_ex_response:
```
We can say `if batch_revoke: {set up logic to revoke all 3 in one call}`, else: go into for loop